openapi/openapi.yaml

```
@@ -508,6 +508,42 @@ components:
         - id
         - title
         - abbr
+    CapitalCommitment:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A uuid used to refer to the capital commitment. 
+          format: uuid
+        type:
+          type: string
+          description: A four character string used to refer to the commitment type.
+          pattern: '^([A-z]{4})$'
+          example: DSGN
+        plannedDate:
+          type: date
+          description: A string used to refer to the date when the commitment is projected to be committed.
+          example: '2012-04-23T18:25:43.511Z'
```

The [date schema documentation](Also , we're likely not going to be returning times with it. ) also made me realize that won't be returning times with the dates.

```suggestion
          example: '2012-04-23'
```